### PR TITLE
add banners w SL / metrics updates link

### PR DESCRIPTION
**Pull request by mirnawong1**
adding callout to share SL future blog post

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/build/metrics.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ keywords:
   - dbt metrics layer
 ---
 
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions, efficient querying, and changes to the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics). For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog.
```
Comment (callum-mcdata) [inline website/docs/docs/build/metrics.md]: Hmm. I think we need more stark language here because we'll be deprecating the dbt_metrics package.

Comment (mirnawong1) [reply website/docs/docs/build/metrics.md]: ok gotcha! i went too conservative! how about this @callum-mcdata :

> The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.

Comment (callum-mcdata) [reply website/docs/docs/build/metrics.md]: > The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info ....

How about that?

Comment (mirnawong1) [reply website/docs/docs/build/metrics.md]: nicee! sounds great thanks @callum-mcdata !

Comment (mirnawong1) [inline website/docs/docs/build/metrics.md]:
```suggestion
The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.
```

File: website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ description: "Review a wide range of partners you can integrate and query with t
 sidebar_label: "Available integrations"
 ---
 
-# Available integrations
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions, efficient querying, and changes to the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics). For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md]:
```suggestion
The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.
```

File: website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md [hunk 3]
```diff
@@ -5,7 +5,9 @@ description: "Introducing the dbt Semantic Layer"
 sidebar_label: "dbt Semantic Layer"
 ---
 
-<Snippet src="sl-public-preview-banner" />
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions, efficient querying, and changes to the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics). For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md]:
```suggestion
The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.
```

File: website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md [hunk 4]
```diff
@@ -7,7 +7,9 @@ sidebar_label: "Quickstart"
 
 # dbt Semantic Layer quickstart
 
-<Snippet src="sl-public-preview-banner" />
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions, efficient querying, and changes to the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics). For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md]:
```suggestion
The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.
```

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md [hunk 5]
```diff
@@ -5,6 +5,10 @@ description: "You can set up the dbt Semantic Layer in dbt Cloud."
 sidebar_label: "Set up the dbt Semantic Layer"
 ---
 
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions, efficient querying, and changes to the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics). For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]:
```suggestion
The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.
```

File: website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md [hunk 6]
```diff
@@ -6,15 +6,14 @@ description: Learn about partner integration guidelines, roadmap, and connectivi
 
 # dbt Semantic Layer partner integration
 
-:::info Contact us
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions, efficient querying, and changes to the [dbt_metrics package](https://docs.getdbt.com/blog/deprecating-dbt-metrics). For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog.
```
Comment (mirnawong1) [inline website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md]:
```suggestion
The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.
```

**Comment by mirnawong1** [comment]
all callouts are updated now @callum-mcdata - lemme know what you think whenever you have a free moment, no rush!

**Review by callum-mcdata — APPROVED** [review]
LGTM!

**Pull request merged**